apps/condo/domains/organization/schema/ResetOrganizationService.test.js

```diff
@@ -179,4 +202,41 @@ describe('ResetOrganizationService', () => {
 
     })
 
+    test('notification not send to resident after reset organization', async () => {
+        const client = await makeClientWithServiceConsumer()
+        const { property, organization, serviceConsumer, resident } = client
+        const [resource] = await MeterResource.getAll(admin, {})
+
+        const [meter] = await createTestMeter(admin, organization, property, resource, {
+            accountNumber: serviceConsumer.accountNumber,
+            unitName: resident.unitName,
+            verificationDate: dayjs().add(-1, 'year').toISOString(),
+            nextVerificationDate: undefined,
+        })
+
+        await createTestMeterReportingPeriod(admin, organization, {
+            notifyStartDay: 1,
+            notifyEndDay: Number(dayjs().format('DD')),
+        })
+
+        await waitFor(async () => {
+            setAllFeatureFlags(true)
+            const [res] = await _internalScheduleTaskByNameByTestClient(admin, { taskName: 'sendSubmitMeterReadingsPushNotifications' })
+            expect(res.id).toBeDefined()
+        })
```

Comment:
What is this `waitFor` for? You don't need it

Reply:
Just leave `await _internalScheduleTaskByNameByTestClient(admin, { taskName: 'sendSubmitMeterReadingsPushNotifications' })`